lib/samson/repo_provider_status.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+module Samson
+  class RepoProviderStatus
+    CACHE_KEY = name
+
+    class << self
+      def errors
+        (
+          Rails.cache.read(CACHE_KEY) ||
+          ["Add repo_provider_status:60 to PERIODICAL environment variable."]
```

Comment:
Could we add something explaining why users should add this to the environment variable?